library/src/scala/FunctionXXL.scala

```diff
@@ -0,0 +1,17 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2002-2013, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+// GENERATED CODE: DO NOT EDIT. See scala.Function0 for timestamp.
+package scala
+
+/** A function with all parameters grouped in an array. */
+trait FunctionXXL {
+
+  def apply(xs: Array[Object]): Object
```

Comment:
FunctionXXL is only used after erasure, when Any is mapped to Object anyway. So I prefer to be explicit.